cmd/snap/cmd_keys.go

```diff
@@ -66,7 +66,7 @@ func outputJSON(keys []Key) error {
 
 func outputText(keys []Key) error {
 	if len(keys) == 0 {
-		fmt.Fprintf(Stdout, "No keys registered, see `snapcraft create-key`")
+		fmt.Fprintf(Stdout, "No keys registered, see `snapcraft create-key`\n")
```

Comment:
hm. I think in most other places this sort of message goes to stderr. Could you change it to `fmt.Fprintln(Stderr, ...)` instead?